earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -38,9 +43,55 @@ class StorageManaged {
     StorageManaged() : timestamp(0), filesize(0) {}
 };
 
+template<class AssetType, class AssetStorageType> class AssetHandleInterface;
+
+template<class AssetType, class AssetStorageType>
+class AssetSerializerLocalXML
```

Comment:
Minor: maybe ought to move this class to a separate file.

Reply:
Moved out to a new file: AssetSerializer.h. I left the name of the new file generic.